controllers/jobs.js

```diff
@@ -163,6 +203,41 @@ exports.deleteJob = asyncHandler(async (req, res, next) => {
     }
   }
 
+  const rows = await db.one(
+    'UPDATE jobs SET deleted_on=to_timestamp($2 / 1000.0) WHERE job_id = $1 RETURNING *',
+    [req.params.id, Date.now()]
+  );
```

Comment:
Lets standardise to using the `new Date().toLocaleString()` method from javascript to insert directly into PostgreSQL's timestamp fields

```suggestion
  const rows = await db.one(
    'UPDATE jobs SET deleted_on=$2 WHERE job_id = $1 RETURNING *',
    [req.params.id, new Date().toLocaleString()]
  );
```

Reply:
Resolved.